sct_ssh.py

```diff
@@ -284,15 +391,32 @@ def copy_cmd(user, test_id, region, force_use_public_ip, src, dest):
 @click.option("-g", "--group-id", default=None, help="GroupId to use, default to create one base on TestId")
 def attach_test_sg_cmd(user, test_id, region, group_id):
     assert user or test_id
-    instances = select_instance_group(region=region, test_id=test_id, user=user)
+    instances = select_instance_group(region=region, backends=['aws'], test_id=test_id, user=user)
 
     for i in instances:
         aws_region: AwsRegion = AwsRegion(get_region(i))
         instance = aws_region.resource.Instance(i['InstanceId'])
-        click.echo(click.style(f"attaching test SG to {get_tags(i).get('Name', 'N/A')}", fg='green'))
+        click.echo(click.style(f"attaching test SG to {get_name(i)}", fg='green'))
         if group_id:
             group_id_to_add = group_id
         else:
             group_id_to_add = aws_region.provide_sct_test_security_group(get_tags(i).get('TestId', 'N/A')).group_id
         all_sg_ids = list(set([sg['GroupId'] for sg in instance.security_groups] + [group_id_to_add]))
         instance.modify_attribute(Groups=all_sg_ids)
+
+
+@click.command("gce-allow-public", help="Attach test default security group to a group of instances")
+@click.option("-u", "--user", default=None,
+              help="User to search for (RunByUser tag)")
+@click.option("-t", "--test-id", default=None, help="test id to search for")
+def gcp_allow_public(user, test_id):
+    assert user or test_id
+    instances = select_instance_group(backends=['gce'], test_id=test_id, user=user)
+    for i in instances:
+        instances_client, info = get_gce_compute_instances_client()
+        gce_set_tags(instances_client=instances_client,
+                     instance=i,
+                     new_tags=['sct-allow-public'],
```

Comment:
we could add some other tag for opening just ssh (e.g. sct-allow-public-ssh) which would open only ssh

Reply:
it's not just for ssh, also for monitoring and such
I prefer it to use the `sct-allow-public` same as we do for test, yes we can split to to multiple tags, but again the functionally would be align to aws SGs (which we didn't separated per type)

so in general align those commands on both ends, need a bit extra work, which isn't needed for anything else right now